acp/internal/mcpmanager/mcpmanager.go

```diff
@@ -199,10 +199,20 @@ func (m *MCPServerManager) ConnectServer(ctx context.Context, mcpServer *acp.MCP
 			}
 		}
 
+		// Create annotations based on tool properties
+		annotations := &acp.MCPToolAnnotations{
+			Title:           tool.Annotations.Title, // Default title is the tool's name if Title is empty
```

Comment:
```suggestion
			Title:           tool.Annotations.Title,
```

Reply:
i changed this.. empty title is fine that's not our place to push this default in